pkg_building.Rmd

```diff
@@ -63,6 +63,8 @@ We recommend you to use the [`codemetar` package](https://github.com/ropensci/co
 
 - Provide a way for users to opt out of verbosity, preferably at the package level: make message creation dependent on an environment variable or option (like ["usethis.quiet"](https://usethis.r-lib.org/reference/ui.html?q=usethis.quiet#silencing-output) in the usethis package), rather than on a function parameter. The control of messages could be on several levels ("none", "inform", "debug") rather than logical (no messages at all / all messages). Control of verbosity is useful for end users but also in tests. More interesting comments can be found in an [issue of the tidyverse design guide](https://github.com/tidyverse/design/issues/42).
 
+- You can provide translations for your package's messages. The [potools](https://michaelchirico.github.io/potools/) R package can help you with that task.
```

Comment:
Maybe simplify to "provide translations ... with the potools R package"? That would remove repetition.